### PR TITLE
update guava 14->23

### PR DESCRIPTION
**Pull request by everywan**
update guava version.

update func getHostText-> getHost.

I tried locally and successed.

### CODE REVIEW

**Comment by everywan** [comment]
issue: https://github.com/vesoft-inc/nebula-java/issues/62

File: client/pom.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
         <slf4j-log4j12.version>1.7.25</slf4j-log4j12.version>
         <slf4j-api.version>1.7.25</slf4j-api.version>
         <junit.version>4.13.1</junit.version>
-        <guava.version>14.0</guava.version>
+        <guava.version>23.0</guava.version>
```
Comment (Nicole00) [inline client/pom.xml:20]: Thanks for your contribution. And what's the purpose to update guava?

Comment (mz223032) [reply client/pom.xml:20]: I tried it with guava version 26, but it seems that there is something wrong with nebula-flink.

**Review by darionyaphet — APPROVED** [review]

File: client/pom.xml (hunk 1 above)
Comment (everywan) [reply client/pom.xml:20]: sorry but what the error is? you also need replace getHostText->getHost too.
I just tried nebula-tools exchange and it successed.
Forgive my english not so good.

Comment (Nicole00) [reply client/pom.xml:20]: Please indicate whether other users such as exchange、spark-connector、 flink-connector need to make changes.

Comment (mz223032) [reply client/pom.xml:20]: yeah, all it's ok with guava version 23.No modification is required.                                                                                                        but when i upgrade it to 26, there is something else wrong

Comment (everywan) [reply client/pom.xml:20]: Maybe there is other different with version23 and 26, you can check the error detail and try to soloved it. Or post the error here.

**Comment by jamieliu1023** [comment]
@everywan Thanks for your contribution! This is Jamie Liu with Nebula Graph. We have sent you the Contributor certificate issued by the Nebula Graph Community via email to memorize this special moment. Also we'd like to send you a mug to mark this event! Could you please kindly reply the email and let me know where we should ship the mug? Thanks again! Wish a you happy and safe 2021!

**Pull request merged**